COMMENTS.md

```diff
@@ -0,0 +1,29 @@
+## Sprint-0
+
+Most of the lectures for sprint-0 felt pretty easy for the students to digest, only exception being the S0D2L2. But not entirely sure what could we trim, my best guess would be to remove the `box-model` part of the lecture, but I do believe that's worth pointing out
+
+During the lectures I tried to apply the concepts while moving along the slides and seemed to work fine for sprint-0, I feel that showing the styles on its most basic form without the distraction of pretty designs/precreated exercises is better for just getting started and the platform already does a lot of nice looking lectures where they practice this concepts
+
+Would love to hear your thoughts on this,
+
+## Sprint-1
+
+For sprint-1 I followed this same methodology for explaining concepts and so far it has resulted in a lot of interaction/good questions and practical examples, but here's my suggestion for next iterations:
+
+- D1: I created a small example that doesn't use BEM or semantic HTML, and since BEM lecture is short I refactored that file with the help of the students. This is to show both how dificult it's to read a file that doens't follows best conventions.
+
+  - [Practice File](./Lectures/Sprint-1/D1/practice/bem-refactor/index.html)
+  - I simplified it a lot from the version I used on class seeing that it took a bit longer than I thought
+
+- D2:
+
+- For the positioning class I tried following the same methodology of just showing the Positioning elements in real time and going on from their questions, but in retrospective for this one having a better structured example would have been better. I created a basic one to be used on the future
+
+  - [Practice File](./Lectures/Sprint-1/D2/practice/positioning/index.html)
+
+- The VScode class went relatively smooth. One common issue I saw was students having difficulty with opening folders and creating files. Not sure if this is something that we can work or just a matter of practice. Mainly used the slides to make sure I didn't miss any extension that we needed to install or hotkeys, but spend most of the lecture showing the actual VSCode interface
```

Comment:
This is a constant problem, unfortunately. We aim to be completely beginner-friendly, but some of our beginners are lacking basic computer literacy skills. We do link to a computer literacy course in the [welcome lesson](https://tripleten.com/trainer/web-ft/lesson/dbbf2ea5-5492-406c-a287-65c8c17b57ef/?from=program), I don't know how many students actually use it. It might not be a bad idea to mention it in an early lecture, especially for students are are very familiar with working with files